src/Pretzel.Logic/Templating/Context/PathExtensions.cs

```diff
@@ -5,7 +5,7 @@ namespace Pretzel.Logic.Templating.Context
 {
     public static class PathExtensions
     {
-        private static readonly string[] MarkdownFiles = new[] { ".md", ".mdown", ".markdown" };
+        private static readonly string[] MarkdownFiles = new[] { ".md", ".mkd", ".mkdn", ".mdown", ".markdown" };
```

Comment:
Leave it as-is. If someone wants a different extension, they can add it in a PR.